src/main/webapp/WEB-INF/view/register.jsp

```diff
@@ -50,7 +50,11 @@
       <label for="password">Password: </label>
       <br/>
       <input type="password" name="password" id="password">
-      <br/><br/>
+      <br/>
```

Comment:
I ended up just copying Robert's code from the profile page into the registration page, so now it makes more sense to have the attribute "aboutMe" since it can actually be set from the registration page.

Reply:
Great, let's have Robert review this then :) will ping him now